screens/AddMembers/AddMembers.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {View, Text, StyleSheet, Image, FlatList, ActivityIndicator } from 'react-native';
+import Contacts from '../Contacts/Contacts';
+import Background from '../../components/Background';
+import NHSStyle from '../../constants/NHSStyle';
+import Theme from '../../constants/Theme';
+import { SafeAreaView } from 'react-navigation';
+import { TextInput } from 'react-native-paper';
+
+
+export default class AddMembers extends React.Component {
```

Comment:
Can we stick with Functional components;

https://reactjs.org/docs/hooks-intro.html#motivation
https://stackoverflow.com/questions/36097965/when-to-use-es6-class-based-react-components-vs-functional-es6-react-components